apis/v1alpha1/generator.yaml

```diff
@@ -505,6 +505,8 @@ resources:
         template_path: model_package/set_resource_name_as_arn.go.tpl
       sdk_update_pre_build_request:
         template_path: common/sdk_update_pre_build_request.go.tpl
+      sdk_update_post_set_output:
+        code: rm.customSetOutput(&resource{ko})
```

Comment:
In the latest version of the code-generator we can do this with `synced.when.in[]` in the generator config. Check out how we do it in the [EKS controller](https://github.com/aws-controllers-k8s/eks-controller/blob/main/generator.yaml#L106-L110)